### PR TITLE
I os update check

### PR DESCRIPTION
**Pull request by Rockindash**
iOS Update check added to Settings and on launch, Issue #1174

### CODE REVIEW

File: VultisigApp/VultisigApp/View Models/PhoneCheckUpdateViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    func resetData() {
+        showError = false
+        showDetails = false
+    }
+    
+    func checkForUpdates(isAutoCheck: Bool = false) {
+        let currentVersion = currentAppVersion()
+        let bundleID = Bundle.main.bundleIdentifier ?? ""
+        
+        fetchLatestAppStoreVersion(bundleID: bundleID) { latestVersion in
```
Comment (flypaper0) [inline VultisigApp/VultisigApp/View Models/PhoneCheckUpdateViewModel.swift:28]: should that be in viewModel?

Comment (johnnyluo) [reply VultisigApp/VultisigApp/View Models/PhoneCheckUpdateViewModel.swift:28]: Isn't you are looking at a viewmodel?  😅

File: VultisigApp/VultisigApp/Utils/Endpoint.swift [hunk 2]
```diff
@@ -31,6 +31,7 @@ class Endpoint {
     
     static let updateVersionCheck = "https://api.github.com/repos/vultisig/vultisig-ios/releases"
     static let githubMacUpdateBase = "https://github.com/vultisig/vultisig-ios/releases/tag/"
+    static let appStoreLink = "https://apps.apple.com/ca/app/vultisig/id6503023896"
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Utils/Endpoint.swift:34]:
```suggestion
    static let appStoreLink = "https://apps.apple.com/app/vultisig/id6503023896"
```

Comment (johnnyluo) [reply VultisigApp/VultisigApp/Utils/Endpoint.swift:34]: Also we already have a variable call `AppStoreVultisigURL`

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**